src/mocks/aktivitet.js

```diff
@@ -514,55 +502,102 @@ export function opprettAktivitet(pathParams, body) {
         opprettetDato: new Date(),
         lagtInnAv: bruker,
         endretDato: moment().toISOString(),
-        endretAv: null,
+        endretAv: bruker,
         versjon: '1',
         erLestAvBruker: eksternBruker,
+        transaksjonsType: 'OPPRETTET',
         ...body,
     });
     aktiviteter.push(newAktivitet);
+    versjoner.push(newAktivitet);
     return newAktivitet;
 }
 
-export function oppdaterAktivitet({ aktivitetId }, aktivitet) {
-    const oldAktivitet = aktiviteter.find((aktivitet) => aktivitet.id === aktivitetId);
-    Object.assign(oldAktivitet, aktivitet);
-    oldAktivitet.endretDato = moment().toISOString();
-    oldAktivitet.endretAv = bruker;
-    oldAktivitet.lagtInnAv = bruker;
+function doOppdaterInternMockStateOgReturnerNyAktivitet(aktivitetId, nyeAktivitetAttributter) {
+    // Hent den gamle aktiviteten
+    const gammelAktivitet = aktiviteter.find((a) => a.id === aktivitetId);
+    // Merge de nye attributtene inn i den originale aktiviteten
+    const nyAktivitet = Object.assign({}, gammelAktivitet, lagNyVersion(nyeAktivitetAttributter));
```

Comment:
Går det an å gjøre sånn her?
```suggestion
    const nyAktivitet = {...gammelAktivitet, lagNyVersion(nyeAktivitetAttributter)};
```

Reply:
Jeg ville være sikker på at jeg fikk en ny kopi og ikke bare overskrev den gamle referansen. Derfor merger jeg gammelt objekt og nye attributter inn i et nytt objekt. 
Jeg er ikke 100% sikker på hvordan spread-operatoren fungerer, men jeg tror kanskje du ville endt opp med en referanse til den gamle aktiviteten, og da smeller det i historikken.

Reply:
Jeg testet ut litt, og denne varianten her fungerer faktisk i følge intensjonen, som en kloning:
`    const nyAktivitet = {
        ...gammelAktivitet,
        ...lagNyVersion(nyeAktivitetAttributter)
    };`

Reply:
Endret fra Object.assign til spread operator for kloning 👍